style.css

```diff
@@ -128,9 +124,10 @@ pre {
   border-color: #ada;
   background: #efe;
   padding: 0 40px;
-  font-size: 85.714%;
-  line-height: 1.334;
-  font-family: "Courier", monospace;
+  font-size: 76.191%;
```

Comment:
```javascript
> (16 / 21 * 100).toFixed(3) + '%'
'76.190%'
```

Chrome seems to use `Math.floor` in some cases rather than `Math.round`. The extra 0.001% prompts Chrome to use 24px per line for the editable snippets rather than 23px per line. Browsers!